app/controllers/HomeController.scala

```diff
@@ -1,17 +1,32 @@
-package controllers
-
-import javax.inject._
-import play.api._
-import play.api.mvc._
-
-@Singleton
-class HomeController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
-
-  def index() = Action { implicit request: Request[AnyContent] =>
-    Ok(views.html.index())
-  }
-  
-  def hello(name: String) = Action { implicit request: Request[AnyContent] =>
-    Ok(views.html.hello(name))
-  }
-}
+package controllers
+
+import scala.concurrent.Future
+
+import com.mohiva.play.silhouette.api.actions.SecuredRequest
+import com.mohiva.play.silhouette.api.LogoutEvent
+import com.mohiva.play.silhouette.api.Silhouette
+import javax.inject._
+import play.api.mvc._
+
+import utils.auth.DefaultEnv
+
+@Singleton
```

Comment:
Does this need to be singleton?

Reply:
I think should not be per https://github.com/google/guice/wiki/Scopes#choosing-a-scope